### PR TITLE
IDEA-350498 Enable check for Robolectric parameterized

### PR DESCRIPTION
**Pull request by karimai-gg**
... test methods.

### CODE REVIEW

**Comment by HackerMadCat** [comment]
Could you add tests for the Robolectric framework?

Must have unit tests for filters. 
  * `org.jetbrains.plugins.gradle.util.GradleExecutionSettingsUtilTest`

Good to have tests for the test event recognition.
  * `org.jetbrains.plugins.gradle.execution.test.events.SpockGradleTestNavigationTest`

**Comment by HackerMadCat** [comment]
Pushed with commit a34f29bea90cc7ea819ff3ec645f827ad2abc495

File: plugins/gradle/java/src/util/GradleExecutionSettingsUtil.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ fun createTestFilterFrom(location: Location<*>?, psiClass: PsiClass?, psiMethod:
       parameterName = "[*$parameterName*]"
     }
   }
-  else if (psiClass != null && psiClass.isJunit4ParameterizedClass()) {
+  else if (psiClass != null && psiClass.isParameterizedClass()) {
```
[the remark this answers is not in the document]
Comment (HackerMadCat) [reply plugins/gradle/java/src/util/GradleExecutionSettingsUtil.kt:80]: fixed by me

**Review by HackerMadCat — APPROVED** [review]

**Comment by karimai-gg** [comment]
This has been merged in commit 7560ac89ce8588ae3451a7c6beac81c6301a9327.

**Pull request closed without merging**